dmutils/documents.py

```diff
@@ -0,0 +1,136 @@
+import os
+import datetime
+
+try:
+    import urlparse
+except ImportError:
+    import urllib.parse as urlparse
+
+from .s3 import S3ResponseError
+
+
+def filter_empty_files(files):
+    """Removes any empty files from the list.
```

Comment:
[PEP257](https://www.python.org/dev/peps/pep-0257/) says this should be written as "Remove any empty files from the list." :trollface: